src/doom/p_inter.c

```diff
@@ -59,14 +59,16 @@ int	clipammo[NUMAMMO] = {10, 4, 20, 1};
 // P_GiveAmmo
 // Num is the number of clip loads,
 // not the individual count (0= 1/2 clip).
+// [NS] Dropped ammo/weapons give half as much.
```

Comment:
This comment should rather go right next to the `dropped` parameter in the function definiton. It gets lost here.